packages/skeleton-loader/src/CardSkeleton/CardSkeleton.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import Card from '@leafygreen-ui/card';
+import { cx } from '@leafygreen-ui/emotion';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+
+import { ParagraphSkeleton } from '..';
+
+import { rootStyles } from './CardSkeleton.styles';
+import { CardSkeletonProps } from '.';
+
+export function CardSkeleton({
+  darkMode: darkModeProp,
+  className,
+  ...rest
+}: CardSkeletonProps) {
+  const { darkMode } = useDarkMode(darkModeProp);
+  return (
+    <Card
+      aria-busy
```

Comment:
There are so many conflicting articles about `aria-busy`. This [article I found](https://adrianroselli.com/2020/11/more-accessible-skeletons.html) shows that it should not be on the skeleton loader but rather the component's parent using it. I'm not sure what's correct but maybe we should also add `aria-hidden` to this?

Reply:
That's a bit tricky to achieve without actually baking the Skeleton into each component, since we would want a flip of the aria-busy attribute from false to true on the container element when the content loads. By having `aria-busy` on the component, we would at least make sure something on the screen reader indicates that the page is loading. In a separate component, we can't really know what the appropriate component to pass `aria-busy` to would be.

Maybe Skeletons shouldn't have aria-busy when they're a part of a larger Skeleton component. But this means that the Card in `CardSkeleton` should have aria-busy and the Skeletons inside shouldn't?

Reply:
To me, it seems that the `aria-busy` should be left up to the consumer to add and toggle, but i doubt anyone would even use it :(

Reply:
But maybe I should add aria-hidden to the basic Skeleton. @bruugey do you know what the best move here would be